rollup.config.js

```diff
@@ -1,25 +1,45 @@
 import { terser as terserPlugin } from "rollup-plugin-terser";
+import resolvePlugin from "@rollup/plugin-node-resolve";
+import commonjsPlugin from "@rollup/plugin-commonjs";
 
 export default ({ configTest }) => {
     const noMinify = Boolean(configTest || process.env.CI);
-    return {
-        input: "main.js",
-        plugins: noMinify ? [] : [
-            terserPlugin({
-                compress: true,
-                mangle: true
-            })
-        ],
-        output: {
-            file: "dist/bundle.min.js",
-            format: "umd",
-            globals: {
-                "source-map": "sourceMap",
+    return [
+        {
+            input: "main.js",
+            plugins: noMinify ? [] : [
```

Comment:
I think we do not need minify nodejs bundle for better debugging experience (see #680)

cc @fabiosantoscode